core/api/common_api.hpp

```diff
@@ -5,22 +5,45 @@
 
 #pragma once
 
+#include <jwt-cpp/jwt.h>
 #include <libp2p/peer/peer_info.hpp>
 
 #include "api/utils.hpp"
 #include "api/version.hpp"
 #include "common/buffer.hpp"
+#include "common/logger.hpp"
+#include "common/span.hpp"
 
 namespace fc::api {
   using common::Buffer;
   using libp2p::peer::PeerInfo;
+  using ApiAlgorithm = jwt::algorithm::hmacsha;
+  const std::string kTokenType = "JWT";
```

Comment:
maybe use `constexpr std::string_view` for string consts.

otherwise duplicate symbols error will occur, if not made `static` or `extern`.

also may move these jwt consts to other header, for those who will actually need/use them.